pkgs/servers/matrix-conduit/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchFromGitLab, rustPlatform, }:
+rustPlatform.buildRustPackage rec {
+  pname = "matrix-conduit";
+  version = "unstable-2021-05-21";
+
+  src = fetchFromGitLab {
+    owner = "famedly";
+    repo = pname;
```

Comment:
The pname is `matrix-conduit` but the gitlab repo is just `conduit`.

Reply:
Right, that one got introduced in the name switch and didn't show up as an error because we've already fetched.